ASP.NET API/WAVC_WebApi/FriendsManager.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading.Tasks;
+using WAVC_WebApi.Models;
+
+namespace WAVC_WebApi
+{
+    public class FriendsManager
```

Comment:
What's the purpose of this class? Controller does the same things. Code duplication.

Reply:
Controller and Hub both use it, so I believe it should be seperate from them.